Python/fitnesse/html_util.py

```diff
@@ -5,38 +5,43 @@
 
 class HtmlUtil:
 
+    wiki_page: WikiPage
+    string_io: StringIO
+    page_crawler: PageCrawlerImpl
+    path_parser: PathParser
+    
+
     @staticmethod
     def testable_html(page_data: PageData, include_suite_setup: bool, page_crawler: PageCrawlerImpl, path_parser: PathParser) -> str:
-        wiki_page: WikiPage = page_data.get_wiki_page()
-        string_io: StringIO = StringIO()
+        HtmlUtil.wiki_page = page_data.get_wiki_page()
+        HtmlUtil.string_io = StringIO()
```

Comment:
It might be better to leave string_io a local object within the function `testable_html` so that the step-by-step update of string_io is explicit within `testable_html`. I hardly imagine it being shared with other functions in the class.